client/src/components/exercise/test-components/Main.jsx

```diff
@@ -1,14 +1,18 @@
 import React from "react";
 import NavigationPane from "../../all-components/Lab/NavigationPane";
 import LabScreen from "../../all-components/Lab/LabScreen";
+import Carousel from "../../all-components/carousel.js";
 
 const title = "Sample Lab Name Here This Long";
 
 const Main = () => {
   return (
-    <div className={"tw-flex tw-mx-12"}>
-      <NavigationPane title={title} />
-      <LabScreen />
+    <div>
+      <div className={"tw-flex tw-mx-12"}>
+        <NavigationPane title={title} />
+        <LabScreen />
+      </div>
+      <Carousel />
```

Comment:
The only thing I'll say is when we put the carousel on the home page it needs a title above saying 'participating schools':
![Screenshot 2024-11-01 at 12 31 36 PM](https://github.com/user-attachments/assets/bd8cf2af-ca1b-4288-8f1b-ab48c4f5c982)